.github/workflows/advance-super.yml

```diff
@@ -34,22 +34,19 @@ jobs:
           # that caused it to wait, reducing push failures down below.
           ref: ${{ github.ref }}
           # We need a token with permission to push.
-          token: ${{ secrets.ZQ_UPDATE_PAT }}
+          token: ${{ secrets.PAT_TOKEN }}
```

Comment:
That Actions secret called `ZQ_UPDATE_PAT` hasn't been updated in 4 years and this is the only Workflow that's been using it. Since it has "ZQ" in its name I'm inclined to get rid of it, and since we have another secret called `PAT_TOKEN` that's used in other Workflows and that one is only 2 years old, I'm game to give that one a shot on the assumption it probably has the appropriate permissions. Once we see this Workflow run successfully with `PAT_TOKEN` I'll delete the `ZQ_UPDATE_PAT` secret from the GitHub config.